stories/imdb/stories/greyVest.js

```diff
@@ -167,11 +170,20 @@ let schemas = fromPromise(
     )
     .then(_.tap(() => tree.refresh(['root'])))
 )
+let input
+let select
+let textArea
 
 export default () => (
   <div className="gv-body">
     <Fonts />
     <GVStyle />
+    <Input ref={e => (input = e)} />
+    <Textarea ref={e => (textArea = e)} />
+    <Select ref={e => (select = e)} />
```

Comment:
This should probably be its own story so that the grey vest imdb story is as close to a real search page as possible